packages/edit-site/src/style.scss

```diff
@@ -56,19 +56,6 @@ body.toplevel_page_gutenberg-edit-site {
 	.interface-interface-skeleton__content {
 		background-color: $light-gray-700;
 	}
-
-	// The button element easily inherits styles that are meant for the editor style.
-	// These rules enhance the specificity to reduce that inheritance.
-	// This is duplicated in visual-editor.
-	& .components-button {
```

Comment:
These styles are actually needed, they're  just being applied  on the wrong selector which i'm fixing here https://github.com/WordPress/gutenberg/pull/23464

Reply:
@youknowriad Ah! I see. What would you recommend I do with this PR?

Reply:
Maybe keep this  PR about the inserter toggle?

Reply:
@youknowriad The button styles were part of that. They bled into all sorts of things. I can revert the changes to `edit-site/src/style.scss` to avoid conflicts. Would that work?

Reply:
yes, so I merged the other branch, you can now rebase this and adapt it.